### PR TITLE
Feature/39446 - login page

### PR DESCRIPTION
**Pull request by szafran89**
Internal: 39446

### CODE REVIEW

File: components/04-views/login/_login.scss [hunk 1]
```diff
@@ -1,170 +1,123 @@
...
-        flex-wrap: nowrap;
+        flex-wrap: wrap;
         justify-content: flex-start;
         align-items: center;
         margin: $login__actions-margin;
     }
+
     &__button {
+        width: 100%;
+        flex: 0 0 100%;
```
Comment (Aekal) [inline components/04-views/login/_login.scss]: Do we need both `flex-basis` and `width` here? I think you can remove `width`, `flex` is enought

File: components/04-views/login/_login.scss [hunk 2]
```diff
@@ -1,170 +1,123 @@
...
+        justify-content: center;
         margin: $login__button-margin;
         padding-right: $login__horizontal-padding;
         padding-left: $login__horizontal-padding;
         min-width: $login__button-min-width;
+        @include mq($screen-s) {
+            width: auto;
+            flex: 0 0 auto;
```
Comment (Aekal) [inline components/04-views/login/_login.scss]: Same here

File: components/04-views/login/_login.scss [hunk 3]
```diff
@@ -1,170 +1,123 @@
...
+        }
+
+        &--create-accunt {
+            margin: $login__button-margin--crate-account;
+        }
     }
+
     &__forgot-password {
-        margin: $login__forgot-password-margin;
+        margin-top: $spacer--medium;
```
Comment (Aekal) [inline components/04-views/login/_login.scss]: Please move to variables

File: components/04-views/login/_login.scss (hunk 1 above)
Comment (szafran89) [reply components/04-views/login/_login.scss]: I have two cases and for one is needed.

File: components/04-views/login/_login.scss (hunk 2 above)
Comment (szafran89) [reply components/04-views/login/_login.scss]: The same

File: components/04-views/login/_login.scss (hunk 3 above)
Comment (szafran89) [reply components/04-views/login/_login.scss]: changed.

File: components/04-views/login/_login.scss (hunk 1 above)
Comment (Aekal) [reply components/04-views/login/_login.scss]: Ok

**Pull request merged**